site/content/contribute/mobile/e2e/android.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Setup and Run Android E2E"
+date: 2020-09-01T09:00:00-00:00
+weight: 3
+subsection: Mobile End-to-End Testing
+---
+
+#### Local setup
+
+1. Install the latest SDK.
+
+```
+sdkmanager "system-images;android-30;google_apis;x86"
+sdkmanager --licenses
+```
+2. Create the emulator using `npm run e2e:android-create-emulator` from the `/detox` folder. Android testing requires an emulator named `detox_emu_api_30` and the script helps to create it automatically.
+
+#### Test run in debug mode
+
+This is the typical flow for local development and test writing:
+
+1. Open a terminal window and run `make run-android` from the root folder.
```

Comment:
is it required to build and run the app first? if so, why? I think this step is not required at all.

Reply:
Ah, right. This works too but actually the minimum requirement is to start react-native packager.  I'll change this with `make start`.

Reply:
Please use `npm start` instead.. I’m going to get rid of the **make** commands

Reply:
Done.